run.sh

```diff
@@ -0,0 +1,7 @@
+docker login
```

Comment:
The user will not need to log in to Docker every time the script is run. I think that we should leave the instruction in the README on how to log in but take it out of the bash script.

Reply:
Oh durr.🤪 Thank you for catching that!